futurenhs.automation/features/search.feature

```diff
@@ -133,7 +125,6 @@ Feature: Search
 
     Scenario: FNHS108 - Search for a Group By the Strapline
         When I search for 'DO NOT USE - This group is reserved solely for use by our automated test scripts'
-        Then there are '6' search results displayed
```

Comment:
I presume you have decided to remove the result count due to instability on data?